### PR TITLE
API & ABI cleanup

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

**Comment by mbouron** [comment]
While at it, can we remove the motion vectors from the API ?

**Comment by ubitux** [comment]
> While at it, can we remove the motion vectors from the API ?

Yeah I have them in my TODO list along with `pkt_skip_mod`. Do you see anything else? Should we deprecate them or use the opportunity of a new project?

Edit: done for both

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `skip` and `trim_duration` options
 - `export_mvs` option
 - `pkt_skip_mod` option
+- `nmd_set_drop_ref()` (was disabled)
```
Comment (mbouron) [inline CHANGELOG.md]: `was disabled` -> `unimplemented` (and same goes to the commit msg) ?

**Review by mbouron — APPROVED** [review]
LGTM, thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (ubitux) [reply CHANGELOG.md]: sure, done

**Review by mbouron — APPROVED** [review]

**Pull request merged**